core/modules/parsers/wikiparser/rules/syslink.js

```diff
@@ -0,0 +1,45 @@
+/*\
+title: $:/core/modules/parsers/wikiparser/rules/syslink.js
+type: application/javascript
+module-type: wikirule
+
+Wiki text inline rule for system tiddler links.
+Can be suppressed preceding them with `~`.
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+exports.name = "syslink";
+exports.types = {inline: true};
+
+exports.init = function(parser) {
+	this.parser = parser;
+	// Regexp to match
+	this.matchRegExp = /~?(?:\$):[^\s<>{}\[\]`|'"\\^~]+(?:\/|\b)/mg;
```

Comment:
Do you mean the rendering of the tiddler title from `$:/core/ui/ViewTemplate/title`? I don't understand how that could be affected by the regexp used here.